Composer/packages/client/src/recoilModel/atoms/botState.ts

```diff
@@ -47,6 +47,22 @@ const emptyDialog: DialogInfo = {
   skills: [],
   isFormDialog: false,
 };
+type lgStateParams = { projectId: string; lgFileId: string };
```

Comment:
PascalCase type

Reply:
How does this combination key work. Does recoil combine projectId and lgFileId into a single key on the back?

Reply:
They serialize the keys